content/docs/command-reference/version.md

```diff
@@ -36,36 +34,36 @@ usage: dvc version [-h] [-q | -v]
 
 The detail of DVC version depends upon the way of installing DVC.
 
-- **Official release**: [These instructions](/doc/install) include ways to
-  install DVC using the official package stored in Python Packaging Authority.
-  We mark these official releases with tags on DVC's repository. Any issues
-  reported with the official build can be traced using the `_BASE_VERSION`
-  constant
-  [in our core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
-  For example `0.40.2`.
+- **Official release**: official releases (for example `1.7.2`) are stable
+  versions that have upgraded features and bug fixes.
+  [These instructions](/doc/install) include ways to install DVC using the
+  official package stored in [Python Packaging Authority](https://www.pypa.io/).
+  We mark these official releases with
+  [tags](https://github.com/iterative/dvc/releases) on DVC's repository. Any
+  issues reported with the official build can be traced using the
+  `_BASE_VERSION` constant in our
+  [core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
```

Comment:
> Any issues reported...

Again, TBH I don't get what this is talking about. Why are we talking about issues? What is this trying to say? I honestly am not sure. Let's always try to fully understand the text we're working with when doing holistic rewrites like this one. We can ask the team on Discord if needed.

Reply:
> I honestly am not sure.

My guess is that since `dvc version` is required to formally report bugs, etc. on the core repo, they wanted to say something about that but is it really relevant to know about _BASE_VERSION to the end-user? Prob not. Is there anything to save about this though, and should it be in the same paragraph? Thanks

Reply:
I think `_BASE_VERSION` is explained here because it's used again in next part (Development version). I have made it as a new paragraph. 
We can actually move this explanation to Development version point and use a simple explanation here like: "report issue with version displayed in output....."

Reply:
Thanks. I still think we don't need to talk about _BASE_VERSION at all, in neither place. Let's just remove it completely? You can substitute for "official version number" in the **Development version** section.